File: java/07 - Demo 3 - Refactoring/src/main/java/renamemethod/Person.java
package renamemethod;

public class Person {
	private String officeAreaCode;
	private String officeNumber;

	Person(String officeAreaCode, String officeNumber) {
		this.officeAreaCode = officeAreaCode;
		this.officeNumber = officeNumber;
	}
	
	public String getNumber() {
		return ("(" + officeAreaCode + ") " + officeNumber);
	}
	

}

File: csharp/07 - Demo 3 - Refactoring/Refactoring/RenameMethod/Person.cs
namespace Refactoring.RenameMethod
{
    public class Person
    {
        private string _officeAreaCode;
        private string _officeNumber;

        public Person(string officeAreaCode, string officeNumber)
        {
            _officeAreaCode = officeAreaCode;
            _officeNumber = officeNumber;
        }

        public string TelephoneNumber => $"({_officeAreaCode }) {_officeNumber}";
        

    }
}